adopters.geojson

```diff
@@ -1,6 +1,7 @@
 {
   "type":"FeatureCollection",
   "features":[
+    {"type": "Feature", "properties": {"institution": "Fontaine's city hall"}, "geometry": {"type": "Point", "coordinates": [5.6882042754225745.688204275422574, 45.19183865]}},
```

Comment:
I am not sure what happend here, but there shouldn't be an additional dot in the first coordinate. Could you please change this to a valid format?

Reply:
Oups, edited too quickly without double check, sorry.